### DISCUSSION TITLE
Help Ubuntu backport a fix to 24.04 LTS

### DISCUSSION THREAD

**Discussion by jeroen** [curl/curl#15272] [General]
Not sure if this is the right forum to discuss, but hopefully the right people might take notice...

Users of the R bindings are complaining about crashes for FTP downloads in the latest Ubuntu 24.04 LTS, which has libcurl 8.5.0. We can reproduce the crash on the command line as well:

```sh
URL1="https://en.wikipedia.org/wiki/Bioconductor"
URL2="ftp://ftp.ensembl.org/pub/release-71/gtf/caenorhabditis_elegans/Caenorhabditis_elegans.WBcel235.71.gtf.gz"
curl --keepalive-time 60 -OL $URL1 -OL $URL2 -OL $URL2
```

I think this is the same bug as #13731 which was fixed in libcurl 8.8.0. Given that this distro will be around for 5 more years, can we try to help them to prepare a patch?

### curl/libcurl version

curl 8.5.0

### operating system

Ubuntu 24.04 LTS

**Comment by ValdikSS** [comment]
You should report this issue to your distro (Ubuntu): https://help.ubuntu.com/stable/ubuntu-help/report-ubuntu-bug.html.en

**Comment by colinleroy** [comment]
I've hit the same problem and reported it at https://bugs.launchpad.net/ubuntu/+source/curl/+bug/2104163

**Comment by vszakats** [comment]
It's often possible to switch over to HTTPS endpoints, e.g.:
```
https://ftp.ensembl.org/pub/release-71/gtf/caenorhabditis_elegans/Caenorhabditis_elegans.WBcel235.71.gtf.gz
```
This isn't a fix, but by avoiding FTP, it can mitigate some cases till a patch lands.

**Comment by bagder** [comment]
I would expect them to reach out to us if they need help. They know where we are.

**Reply by colinleroy** [reply]
Given my previous experience with ubuntu lts bugfixes, they're not going to do much about it unless there's a cve attached.
(My previous experience is spending three hours preparing a nice report with a debdiff for a Gimp bug, getting told they can't reproduce, then watching the silence once someone reproduced)
